api/dhcp/v1alpha2/dhcpsubnet_types.go

```diff
@@ -38,6 +38,14 @@ type DHCPSubnetSpec struct {
 	CircuitID string `json:"circuitID"`
 	// PXEURL (optional) to identify the pxe server to use to boot hosts connected to this segment such as http://10.10.10.99/bootfilename or tftp://10.10.10.99/bootfilename, http query strings are not supported
 	PXEURL string `json:"pxeURL"`
+	// DNS server address (optional) to configure Domain Name Serverfor this particular segment such as 10.10.10.2
+	DNSServer string `json:"dnsServer"`
```

Comment:
DNSServer should be `DNSServers []string` as in most cases you want to pass multiple DNS servers. Same for the VPC API.